.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Build manage-frontend
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  CI:
+    if: >-
+      (github.actor != 'dependabot[bot]') &&
```

Comment:
Curious to see a filtering out of Dependabot PRs. Does the team have a particular process for reviewing Dependabot PRs?

Reply:
This particular bit of filtering is some boilerplate I used in Conversions, but may not be relevant to Retention. @johnduffell, @rupertbates thoughts?

Reply:
I've removed this filter, as I don't think it's needed/wanted for this repo.